src/types/simple.rs

```diff
@@ -364,17 +316,6 @@ impl TypeRow {
         }
     }
 
-    /// Create a new row from a Cow slice of types.
-    ///
-    /// See [`type_row!`] for a more ergonomic way to create a statically allocated rows.
-    ///
-    /// [`type_row!`]: crate::macros::type_row.
-    pub fn from(types: impl Into<Cow<'static, [SimpleType]>>) -> Self {
```

Comment:
why is this method gone?

Reply:
Because we already
```
impl<T> From<T> for TypeRow
where
    T: Into<Cow<'static, [SimpleType]>>,
```
and AFAICT this wasn't doing anything. (I removed it first in its own commit and everything was happy.)